plugins/flexibleengine/pom.xml

```diff
@@ -93,10 +93,10 @@
             <version>${huaweicloud.sdk.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
+            <groupId>ch.qos.logback</groupId>
```

Comment:
@baixinsui do we really need to have logback here as dependency? i thought having the SL4J as bridge should be sufficient.

Reply:
Restore.